core/src/server/rpc_module.rs

```diff
@@ -205,6 +205,18 @@ impl Methods {
 		Ok(())
 	}
 
+	/// Merge two [`Methods`]'s by adding all [`MethodCallback`]s from `other` into `self`.
+	///
+	/// Unlike [Self::merge], existing callbacks are replaced.
```

Comment:
would be good with a rustdoc links here, not sure if `Self::merge` works then

```suggestion
	/// Unlike [`Methods::merge`], existing callbacks are replaced.
```